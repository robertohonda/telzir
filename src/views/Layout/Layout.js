import React from 'react'
import AppBar from '../../components/AppBar/AppBar'
import CssBaseline from '@material-ui/core/CssBaseline'
import { makeStyles } from '@material-ui/core/styles'

const useStyles = makeStyles((theme) => ({
  main: {
    paddingTop: 56 + theme.spacing(2),
    padding: theme.spacing(2),
    height: '100%',
    backgroundColor: 'white',
    [theme.breakpoints.up('sm')]: {
      paddingTop: 64 + theme.spacing(2),
    },
  },
}))

const Layout = (props) => {
  const { renderRoutes, routes } = props
  const classes = useStyles()
  return (
    <>
      <CssBaseline />
      <AppBar logoPath="logo.png" title="Telzir" />
      <main className={classes.main}>{renderRoutes(routes)}</main>
    </>
  )
}

export default Layout
