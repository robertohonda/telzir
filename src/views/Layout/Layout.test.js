import React from 'react'
import { shallow } from 'enzyme'
import Layout from './Layout'

describe('Layout', () => {
  it('should render correctly', () => {
    const component = shallow(<Layout renderRoutes={() => {}} />)

    expect(component).toMatchSnapshot()
  })
})
