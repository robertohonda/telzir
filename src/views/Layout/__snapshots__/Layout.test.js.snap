// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render correctly 1`] = `
<Fragment>
  <CssBaseline />
  <CustomAppBar
    logoPath="logo.png"
    title="Telzir"
  />
  <main
    className="makeStyles-main-1"
  />
</Fragment>
`;
